### ISSUE TITLE
NVDA Feedback: missing arrow key commands in tests 7 through 10  in Radio Group Example Using Roving tabindex, V25.02.12

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1242]
## Description of Behavior

JAWS and VO have arrow key commands for tests 7 through 10 but NVDA does not. I thought it might be because NVDA was reading all radios on the same line, but that doesn't appear to be the case, at least not in the current build. In the tests for requesting information about a radio, insert+up is used and is reading only one radio.

## Test Setup

- Test File: [test-20-navToNextUncheckedRadio-nvda.collected.html](https://aria-at.netlify.app/tests/radiogroup-roving-tabindex/test-20-navToNextUncheckedRadio-nvda.collected.html)
- Report Page: [Link](https://aria-at.w3.org/report/163636/targets/356#result-NmJlOeyIxMiI6ODM0fQGEwMD)
- AT: NVDA (Version 2024.4.1)
- Browser: Chrome (Version 133)
